File: app/src/main/java/com/kee0kai/thekey/utils/TimeFormats.java
package com.kee0kai.thekey.utils;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.TimeZone;

public class TimeFormats {

    public static DateFormat simpleDateFormat() {
        DateFormat formatter = SimpleDateFormat.getDateInstance();
        formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
        return formatter;
    }



}

File: app_mobile/src/main/kotlin/com/github/klee0kai/thekey/app/utils/common/TimeFormats.kt
package com.github.klee0kai.thekey.app.utils.common

import java.text.DateFormat
import java.text.SimpleDateFormat
import java.util.TimeZone

object TimeFormats {

    fun simpleDateFormat(): DateFormat {
        val formatter = SimpleDateFormat.getDateInstance()
        formatter.timeZone = TimeZone.getTimeZone("UTC")
        return formatter
    }

}
